flang/test/Driver/driver-options.f90

```diff
@@ -0,0 +1,6 @@
+! Check support of -m32 and -m64.
```

Comment:
Can the file be renamed to be something specific about the enabled options?

Reply:
Yes, of course